lib/Github/Api/Notification.php

```diff
@@ -57,4 +57,19 @@ public function markRead(DateTime $since = null)
 
         $this->put('/notifications', $parameters);
     }
+    /**
+     * [UNDOCUMENTED] Gets a single notification using his ID
+     *
+     * @link https://developer.github.com/v3/activity/notifications/#view-a-single-thread
+     *
+     * @param ID|integer $id
+     */
+    public function id(integer $int)
```

Comment:
I am having trouble with the ```ìntegrer``` imput type. It should be a number. What sould I put here?